### DISCUSSION TITLE
on continuation of my reload ui problem

### DISCUSSION THREAD

**Discussion by NaotoEnjoyer** [TheLastBen/fast-stable-diffusion#1460] [Q&A]
hey i just noticed that when i click on install update repo it starts to update as usual but right in the end briefly shows some msg that your local something etc etc and instanly closes update log and give me done on this section of notebook.. i guess this is why i have problems it cant update my repo for some reason, and yes i tried removing renaming repo so it does clean install but it gives same error
UPDATE - it seems i managed to do clean install but dude pls i beg of you make git pull fricking optional tick option it literally caused me sd breaks 3 times already and i have to tediously move my stuff to new folder.. that it forces to update each session man, its just bad..

**Comment by TheLastBen** [comment]
comment out the line "git pull"

**Reply by NaotoEnjoyer** [reply]
i mean yeah thats what i do, but i etiher have to do this each new session or save notebook copy and then in the end again resave it once its updated and do same thing again byt removing this line, its rly worth the effort to make git pull as an option brother, its rly dangerous for colab users to gi pulling each launch so carefree i got breaking updates  3 times already and only way to fix such updates is to create clean sd repo copy and then tediously moving your personal files to new repo then deleting old folder its rly poor quality of life situation

**Reply by TheLastBen** [reply]
if you want to contribute, remove, the clear_output() from that cell, keep git pull, and let me know which file prevents the git pull from happening

**Reply by NaotoEnjoyer** [reply]
wait wait maybe my dumbass explained it wrong.. all i wanted to say the only problem i have with colab rn is that i have to manually open code on install repo section and delete git pull line so it wont do it once i execute it when i launch new session, and it would be rly nice if you added git pull optional checkbox on that section thats all, its not good to always git pull on each start since if automatic makes breaking update its pretty much always ends up by fresh repo reinstall and tedious personal files migration

**Reply by TheLastBen** [reply]
I have to add git reset and git pull, otherwise other things will break, at the moment webui.sh is causing the git pull to not work which leads to errors
